examples/article-with-lightbox-2.0-gallery.amp.html

```diff
@@ -340,6 +340,41 @@
 </head>
 
 <body>
+  <amp-analytics>
+      <script type="application/json">
+      {
+        "requests": {
+          "open": "https://foo.com/open",
+          "thumb": "https://foo.com/thumb",
+          "slide": "https://foo.com/slide",
+          "descToggle": "https://foo.com/descToggle",
+          "descExpand": "https://foo.com/descExpand"
+        },
+        "triggers": {
+          "trackLightboxOpened": {
+            "on": "lightboxOpened",
+            "request": "open"
+          },
+          "trackThumbnailsViewToggled": {
+            "on": "thumbnailsViewToggled",
+            "request": "thumb"
+          },
+          "trackSlideChange": {
+            "on": "slideChange",
+            "request": "slide"
+          },
+          "trackDescriptionToggled": {
+            "on": "descriptionToggled",
+            "request": "descToggle"
+          },
+          "trackDescriptionExpanded": {
+            "on": "descriptionExpanded",
+            "request": "descExpand"
+          }
+        }
+      }
+      </script>
+      </amp-analytics>
   <header id="top-bar">
     <button class="menu" title="Toggle Sidebar">
```

Comment:
Can we also add these analytics events in documentation? Preferably include some portion of your working example code here too. =)